rebar.config

```diff
@@ -114,7 +114,7 @@
                     {deps, [{meck, ".*", {git, "git://github.com/eproxus/meck.git", {tag, "0.8.6"}}},
                             {websocket_client, ".*", {git,
                             "git://github.com/aeternity/websocket_client",
-                            {ref, "e88797c"}}}
+                            {ref, "a4fb3db"}}}
```

Comment:
I see https://github.com/aeternity/websocket_client/compare/e88797c...a4fb3db
Still for test only. Ok.